Makefile

```diff
@@ -81,7 +81,7 @@ pyenv${PYTHON_VERSION}:
 	. pyenv${PYTHON_VERSION}/bin/activate && \
 		pip install azdev && \
 		azdev setup -r . && \
-		pip install azure-cli==2.0.81 && \
+		pip install azure-cli && \
```

Comment:
Rather than unpinning, can we pin this to the latest known supported version?

Reply:
Latest version of `azure-cli` doesn't work here:
```
ImportError: cannot import name timezone
Makefile:119: recipe for target 'test-python' failed
make: *** [test-python] Error 1
```

I think the original fix of adding `pip install azure-mgmt-apimanagement==0.1.0` should suffice until we decide how to move forward. This just broke with the latest version of that, `0.2.0`.